hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -124,7 +114,6 @@ def __repr__(self) -> str:
         return f"Status.{self.name}"
 
 
-@dataclass_transform()
 @attr.s(slots=True, frozen=True)
 class StructuralCoverageTag:
```

Comment:
I'm assuming `dataclass_transform` isn't necessary nowadays - hopefully attrs is applying it to all attrs classes themselves?